ui/apps/platform/src/Containers/ConfigManagement/Dashboard/Page.js

```diff
@@ -14,6 +15,14 @@ import SecretsMostUsedAcrossDeployments from './widgets/SecretsMostUsedAcrossDep
 
 const ConfigManagementDashboardPage = () => {
     const [isExporting, setIsExporting] = useState(false);
+    const { hasReadAccess } = usePermissions();
+    const hasReadAccessForPolicyViolationsBySeverity =
+        hasReadAccess('Alert') && hasReadAccess('WorkflowAdministration');
+    const hasReadAccessForComplianceByControls = hasReadAccess('Compliance');
+    const hasReadAccessForUsersWithMostClusterAdminRoles =
```

Comment:
At first I read this as "Most cluster admin roles" being the `'Cluster'` and `'K8sRoleBinding'` resources checked for this value, and thought that was an relatively vague naming for the variable. Then I realized that this naming corresponds to the widget, which makes more sense. 😅